### DISCUSSION TITLE
UIButton still not working , appears in mainstory board but doesn't when I run the app

### DISCUSSION THREAD

**Discussion by sendhelpp** [sst-inc/help#23] [Q&A - iOS] [answered]
what do I need to configure to fix this.
[contact list v3 2.zip](https://github.com/sst-inc/help/files/6939936/contact.list.v3.2.zip)
pls help

**Comment by theboi** [comment, marked as answer]
Hi there, ur issue is that ur Initial View Controller is set to the TableViewController not the NavigationController. Hence the navbar is gone so u hv no button

<img width="753" alt="Screenshot 2021-08-06 at 8 29 52 PM" src="https://user-images.githubusercontent.com/48428085/128510805-54964168-0efa-47b9-8ee9-9fe90cf2d456.png">

To fix this, simply check the "Is Initial View Controller" when the NavigationController is selected

**EDIT: ive included an annotated image**

<img width="753" alt="Screenshot 2021-08-06 at 8 32 24 PM" src="https://user-images.githubusercontent.com/48428085/128511150-424f3ea7-efb4-4add-9373-4d5073ba2867.png">

**Reply by sendhelpp** [reply]
works thanks
